### PR TITLE
Add props and propTypes, update codebase, update example

### PR DESCRIPTION
**Pull request by Raizerz**

### CODE REVIEW

**Comment by matiit** [comment]
@Raizerz 

You sure you want to commit `example/dist/bundle.js` file?

File: chartify.js [hunk 1]
```diff
@@ -620,79 +620,54 @@ return /******/ (function(modules) { // webpackBootstrap
...
 		_inherits(Chartify, _Component);
 
+		_createClass(Chartify, null, [{
+			key: 'propTypes',
+			value: {
+				boxSize: _react.PropTypes.number,
+				data: _react.PropTypes.array,
+				height: _react.PropTypes.number,
+				line: _react.PropTypes.bool,
+				theme: _react.PropTypes.string
```
Comment (matiit) [inline chartify.js]: Could use here the `oneOf` propType maybe?

https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes

I am not sure whether this component is meant to be extensible in any way - in that case `oneOf` wouldn't be sufficient enough, but if the list of themes is const - I'd probably use `oneOf`

**Comment by Raizerz** [comment]
@matiit 
1. I commited it because this one is builded from current version, and it could be used for changing source in dev-mod. But seems it's not necessary to push it on every change.
2. At the moment themes list doesn't exist so I added validation for current version. In the future it will be changed so new validation will be added.

**Comment by kirillstepkin** [comment]
@Raizerz @matiit Thanks, guys!

**Pull request merged**